frontend/src/views/assignments/offer-actions.js

```diff
@@ -58,15 +59,30 @@ function OfferActionButtons(props) {
         setOfferForAssignmentRejected,
     } = props;
 
+    const [
+        ddahDeletionConfirmationVisible,
+        setDdahDeletionConfirmationVisible,
```

Comment:
I personally feel that the names are a bit too long...

Reply:
Maybe something like `showWithdrawConfirm` or similar?

Reply:
`showDeleteConfirmation`